_projects/armv7a7-imx6ull-evk/build.project

```diff
@@ -26,7 +26,16 @@ BOOT_DEVICE="nor0"                # Default boot device
 OFFS_USER_SCRIPT=$((0x20000))     # Disk image user script offset (in bytes)
 
 KERNEL_OFFS=$((0x21000))          # Disk image kernel offset (in bytes)
+ERASE_SIZE=$((16 * 4096))         # Rootfs disk erase block size (in bytes)
+SECTOR_SIZE=$((0x100))            # Rootfs write sector size (in bytes)
+CLEANMARKER_SIZE=$((0x10))        # JFFS2 cleanmarker size (in bytes)
+FLASH_IMAGE=y
```

Comment:
hmm, maybe name it better (`IMAGE_PAD_FF`?)  or at least write a comment what it means (`# disk image: pad with 0xFF (necessary for flash memory)`?)

Edit: maybe more detailed comment about the meaning should be in `b_add2img`